kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/wl-pod-wait.sh

```diff
@@ -46,7 +46,9 @@ function usage() {
                         the domain resource's 'spec.restartVersion'
                       - same 'weblogic.introspectVersion' label value as
                         the domain resource's 'spec.introspectVersion'
-                      - same image as the the domain resource's image
+                      - same image as the domain resource's 'spec.image'
+                      - same common mount images as
+                        the domain resource's 'spec.serverPod.commonMounts'
```

Comment:
Should this be `spec.serverPod.commonMounts.image`?

Reply:
Ignore, I see that it can be multiple common mount images.

Reply:
Oops - missed your 'ignore'. I've reworded it a bit.